lnbits/satspay
# refactor: use internal `mempool.js`

See https://github.com/lnbits/satspay/issues/25

**Test Steps**:
 - create an onchain charge (testnet)
 - make an onchain payment
 - the UI should detect that the payment was mafe
